nominees/public-lab.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "Public Lab",
+  "aliases": [
+    "The Public Laboratory for Open Technology and Science",
+    "PLOTS"
+  ],
+  "description": "Public Lab is a worldwide community that develops and applies open-source tools for environmental exploration and investigation. Public Lab works to build a healthier and more equitable world through community science, open technology, and environmental justice. Public Lab supports local environmental problem owners to draw upon expertise from a distributed global community when crafting research questions, through designing and implementing accessible monitoring techniques, to developing advocacy pathways.",
+  "website": "https://publiclab.org/",
+  "license": [
+    {
+      "spdx": "GPL-3.0",
+      "licenseURL": "https://github.com/publiclab/plots2/blob/main/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 3,
+      "evidenceText": "We work together in public to make environmental health research accessible to all",
+      "evidenceURL": "https://publiclab.org/wiki/start-enviro-monitor-study"
+    },
+    {
+      "SDGNumber": 6,
+      "evidenceText": "We research issues with stormwater and wastewater for both DIY, community-led, and governmental accountability solutions",
+      "evidenceURL": "https://publiclab.org/tag/stormwater"
+    },
+    {
+      "SDGNumber": 13,
+      "evidenceText": "Environmental injustice contributes to climate change. Many \"EJ\" communities already live in a reality where government isn't going to prepare them, isn't going to rescue them, and definitely won't rebuild them from increasing climate impacts. For these reasons our community invests in ",
+      "evidenceURL": "https://publiclab.org/tag/disaster-response"
+    },
+    {
+      "SDGNumber": 15,
+      "evidenceText": "Use a kite, balloon, or pole to take an aerial photo and tell your own visual story of a place. https://publiclab.org/tag/land-use ",
+      "evidenceURL": "https://mapknitter.org/"
+    },
+    {
+      "SDGNumber": 16,
+      "evidenceText": "Community-collected data has the power to prompt local, state, or federal action by alerting government agencies to an issue. Failures and gaps at the regulatory level are often what leads communities to do their own environmental monitoring and advocacy work.",
+      "evidenceURL": "https://publiclab.org/wiki/law-and-policy"
+    }
+  ],
+  "sectors": [
+    "Health",
+    "Climate"
+  ],
+  "type": [
+    "software"
+  ],
+  "repositories": [
+    {
+      "name": "main",
+      "url": "https://github.com/publiclab/plots2"
+    },
+    {
+      "name": "MapKnitter",
+      "url": "https://github.com/publiclab/mapknitter"
+    },
+    {
+      "name": "Spectral Workbench",
+      "url": "https://github.com/publiclab/spectral-workbench.js"
+    },
+    {
+      "name": "Image Sequencer",
+      "url": "https://github.com/publiclab/image-sequencer"
```

Comment:
It is unclear to me whether these should be bundled with PLOTS, split off into their own nomination, or excluded. [MapKnitter](https://github.com/publiclab/mapknitter#mapknitter) is useful but seems more like a developer tool compared to a platform or common software. [spectral-workbench.js](https://github.com/publiclab/spectral-workbench.js#spectral-workbenchjs) could be a software DPG for data analysis (or is that considered more like a developer tool in the DPGA interpretation?). If there is open source data to go along with it, it could be a double DPG (software plus data). [Image Sequencer](https://github.com/publiclab/image-sequencer#image-sequencer) could be a software DPG, but I don't know much about image processing. If it were to be its own nominee, it needs better connection for its role in attaining the SDGs.